src/helpers/parse.hpp

```diff
@@ -166,8 +169,11 @@ inline std::variant<parsed_request_t, default_error_t> split_body_headers(std::s
         if (pos == std::string_view::npos)
             return default_error_t{-32700, "Invalid JSON was received by the server."};
         req.body = body.substr(pos + 4);
-    } else
+        auto res = std::from_chars(req.content_length.begin(), req.content_length.end(), req.json_length);
```

Comment:
The result of parsing error should be handled.

Reply:
We now return the same Invalid JSON error.   We could add a new code "Invalid HTTP request" for the 2 errors here.